pkg/version/version.go

```diff
@@ -55,11 +55,17 @@ func String() string {
 }
 
 func GetBuild() *Build {
+	var b string
+	if BuildNumber == "" {
+		b = "0"
```

Comment:
Just set BuildNumber to "0", no need to have another variable or an else statement